solmanager.py

```diff
@@ -45,64 +44,68 @@
 import SolVersion
 import SolDefines
 
+#============================ logging =========================================
+
+logging.config.fileConfig('logging.conf')
+log = logging.getLogger('solserver')
+log.setLevel(logging.DEBUG)
+
 #============================ defines =========================================
 
 #===== defines
 
-FLOW_DEFAULT                           = 'default'
-FLOW_ON                                = 'on'
-FLOW_OFF                               = 'off'
-
-DEFAULT_CONFIGFILE                     = 'solmanager.config'
-DEFAULT_SERIALPORT                     = 'COM18'
-DEFAULT_TCPPORT                        = 8080
-DEFAULT_FILECOMMITDELAY_S              = 60
+FLOW_DEFAULT                            = 'default'
+FLOW_ON                                 = 'on'
+FLOW_OFF                                = 'off'
 
-DEFAULT_CRASHLOG                       = 'solmanager.crashlog'
-DEFAULT_BACKUPFILE                     = 'solmanager.backup'
+DEFAULT_CONFIGFILE                      = 'solmanager.config'
 
-#===== configuration (TODO: only read from file)
+#===== configuration
 
-DEFAULT_LOGFILE                        = 'solmanager.sol'
-DEFAULT_SOLSERVER                      = 'localhost:8081'
-DEFAULT_SOLSERVERTOKEN                 = 'DEFAULT_SOLSERVERTOKEN'
-DEFAULT_SOLMANAGERTOKEN                = 'DEFAULT_SOLMANAGERTOKEN'
-DEFAULT_SOLMANAGERPRIVKEY              = 'solmanager.ppk'
-DEFAULT_SOLMANAGERCERT                 = 'solmanager.cert'
-DEFAULT_SOLSERVERCERT                  = 'solserver.cert'
-DEFAULT_SENDPERIODMINUTES              = 1
-DEFAULT_FILEPERIODMINUTES              = 1
+LOGFILE                                 = ''

```

Comment:
in general, just use the config from the config file directly, don't populate a global var